Makefile

```diff
@@ -272,11 +273,12 @@ $(INVEST_BINARIES_DIR): | $(DIST_DIR) $(BUILD_DIR)
 # Documentation.
 # API docs are built in build/sphinx and copied to dist/apidocs
 apidocs: $(APIDOCS_TARGET_DIR) $(APIDOCS_ZIP_FILE)
-$(APIDOCS_TARGET_DIR): | $(DIST_DIR)
+
+$(APIDOCS_BUILD_DIR): install  # need contents of build/lib to build apidocs
```

Comment:
Unfortunately `install` runs every time, even if nothing has changed, so it adds a bit of time to `make apidocs`. But I don't think it's a problem.